### PR TITLE
Speed up page rendering time by not using embedded video player

### PR DESCRIPTION
**Pull request by mistermocha**
Following implementation described here: https://www.sitepoint.com/faster-youtube-embeds-javascript/

### CODE REVIEW

File: static/new/css/style.css [hunk 1]
```diff
@@ -332,3 +332,29 @@ section.topics.duplicatable-content.home-page {
...
...+KqOkD+YNfHej1QKD+y7iVlOhgLvFqFfNJvNGyuBJ+KDAF8MDd0tgS8y64OlgSdJMsysL4cG7SOHkyQZLhTee7+d2R2rAVy/S+Jd7/32ouBHAP4gNNRGQyTHc/84NhqNywZp5rvjjnnvt21aABFeCQ+RLwAf2hQ8s7sv9OCLk6AHNgQvIrvbfzKCD76g/O6cu7lf/iER/aQGgy448pExZmhdegAPhR9sObFWH1gT3lp7DaA/5bkIgJhZPgsNmz02novj+KqeApj1ubwXWe4kdyeznAgNvTpE/HQmvKqOMeuFogTUVQSRno+iaLRLAJF7uIgL9O4ubgL8aWgB7S44mNX+35YpICUiAvS9sBLkq1WzT+NFffl6AuoiApi6NT37h6sWkBIRZGkQ8YtLgyji6e1mBYTqCEBPG2Naz+0BWQgtoGoRgCzEsd9hAN1X5BfnFZASUfrSAFQNsyZ1FJASUVpHiLinDJG8U2cBZYogkrcNs5waBAGdstbeU9zdqpw0gPwwSAI6VUxHyFlDpOcHUUBBIuYNs14aZAE5RVwyzPr3/0EAEY0TyfGNjBWQvwZ +CTSbehfAH29mrID8bET0+0EUkAd8WYDOmqJ3ecsG30yr9wqRfm6Y+a1BEFDEjHfHvWmY9ck6CygHvBVr8Xhtb4ZE5HZA3y8DvBNA1TjnrmXWf+sioMwZX5V/VHXMGGMMoKdDCxCRvRWBdzKzdHEO+EisilbPyopHYqp6S9UCAsz4iojI7hUDAtyXVQgIDd6KnOoaWNkbI6FaPSuZGyMArsi7MZoloB4zviI/Nhr3X95jltwTRQmoIfgisy5ai+me67OI7fE4nrqjrqfK1t0eby0FPRB6oGVlchL3rgnfrq19RKbVBdhV9IOSwJmfmJi4vi/4ThERitwyCxVAFqydshuCX5awhQ9KtmuIWd8IDZED/nXT77rvVVv6sHRKwjYi91poqP7Dr+Y6JJ1VSZIMA3wkPNy6bX+o8Bcm0sXMdwM8Fxo0A3xORPaWBp6uPXsmbxCRD0NDL0dOANhVCXy6iAjMcjbcrMt3RITKwdMVRdFo+y5yvkL4eWZ+zHt/ZVD4dEVRNGotpst+dZZZH8k86lqn2pIvT/eqrNfn2xuyqYPZ8mv7s8pfn/8Pybm4TIjanscAAAAASUVORK5CYII=") no-repeat center center;
```
Comment (mistermocha) [inline static/new/css/style.css:345]: FYI this is the base-64 encoded play button. Rendering the content up front like this loads it way faster and doesn't consume another TCP socket.

https://www.iconfinder.com/icons/118620/play_icon#size=48

**Comment by mistermocha** [comment]
This fixes issue https://github.com/pybay/pybay/issues/60

File: static/new/css/style.css [hunk 2]
```diff
@@ -337,8 +337,8 @@ section.topics.duplicatable-content.home-page {
     background-position: center;
     background-repeat: no-repeat;
     position: relative;
-    display: inline-block;
-    overflow: hidden;
+    /*display: inline-block;
```
Comment (mzdaniel) [inline static/new/css/style.css]: Do we need to leave this css here?

File: static/new/css/style.css (hunk 1 above)
Comment (mzdaniel) [reply static/new/css/style.css:345]: sure

File: static/new/css/style.css (hunk 2 above)
Comment (mzdaniel) [reply static/new/css/style.css]: Hey Bryan, This PR looks great in http://staging.pybay.com. Lets resolve the css comment and squash with a sensible commit message, and we'll be ready to merge and push to prod.

**Comment by mistermocha** [comment]
Addressed. Sorry about that!
          
            
            On May 22 2017, at 6:46 pm, Daniel Mizyrycki <notifications@github.com> wrote:
            
            
              @mzdaniel commented on this pull request.



In static/new/css/style.css:
> @@ -337,8 +337,8 @@ section.topics.duplicatable-content.home-page {
     background-position: center;
     background-repeat: no-repeat;
     position: relative;
-    display: inline-block;
-    overflow: hidden;
+    /*display: inline-block;

Hey Bryan, This PR looks great in http://staging.pybay.com. Lets resolve the css comment and squash with a sensible commit message, and we'll be ready to merge and push to prod.

**Comment by mzdaniel** [comment]
Excellent. Thank you Brian

**Pull request merged**